tools/test.py

```diff
@@ -65,6 +65,14 @@ def cmp(x, y):  # Python 3
 except NameError:
   xrange = range    # Python 3
 
+try:
+  import urllib.parse                      # Python 3
+  urllib_unquote = urllib.parse.unquote
+except ImportError:
+  import urllib                            # Python 2
+  urllib_unquote = urllib.unquote
+
```

Comment:
Simplify?
```python
try:
    from urllib.parse import unquote as urllib_unquote
except ImportError:
    from urllib import unquote as urllib_unquote
```

Reply:
Or leave off the ___as___ part and just call __unquote()__ below.

Reply:
Thank you 🙂 I updated the PR with this suggestion now.